index.html

```diff
@@ -13435,7 +13436,7 @@ <h3>State and Property Attributes</h3>
 	</section>
 	<section id="host_general_focus">
 		<h2>Focus Navigation</h2>
-		<p>An implementing host language MUST provide support for the author to make all interactive elements focusable, that is, any renderable or event-receiving elements. An implementing host language MUST provide a facility to allow web authors to define whether these focusable, interactive elements appear in the default tab navigation order. The <code>tabindex</code> <a>attribute</a> in <abbr title="Hypertext Markup Language">HTML</abbr> is an example of one implementation.</p>
+		<p>An implementing host language MUST provide support for the author to make all interactive elements focusable, that is, any renderable or event-receiving elements. An implementing host language MUST provide a facility to allow web authors to define whether these focusable, interactive elements appear in the default tab navigation order. The <code>tabindex</code> <a>attribute</a> in HTML is an example of one implementation.</p>
```

Comment:
<abbr title="Today I learned">TIL</abbr> that when ReSpec encounters a unique `<abbr>` pattern in the document, it [automatically generates identical `<abbr>` markup for all subsequent occurrences](https://respec.org/docs/#abbreviations:~:text=ReSpec%20will%20then%20wrap%20all%20matching%20instances%20abbreviations%20with%20%3Cabbr%3E.) of the plain text abbreviation string. 🤩 

Because I encountered a few explicit abbreviation markups for “HTML” during this PR, I found and left the first occurrence of “`<abbr title="Hypertext Markup Language">HTML</abbr>`” [on line 475](https://github.com/w3c/aria/blob/06b266f0570c6319c1b5ea4dee316940062ea5ce/index.html#L475), then simplified all later occurrences to just plain “HTML” to reduce noise in the document.

<abbr title="For what it’s worth">FWIW</abbr>, it looks like we’ve got a ton more abbreviations we could simplify if that were desirable?  E.g., “WAI-ARIA”, “DOM”, “API”.  If anyone think that’s a good idea, please say the word — I’d be happy to do a separate PR for that.

*[TIL]: Today I learned
*[FWIW]: For what it’s worth